src/systemcmds/ver/ver.cpp

```diff
@@ -76,6 +77,7 @@ static void usage(const char *reason)
 	PRINT_MODULE_USAGE_COMMAND_DESCR("bdate", "Build date and time");
 	PRINT_MODULE_USAGE_COMMAND_DESCR("px4guid", "PX4 GUID");
 	PRINT_MODULE_USAGE_COMMAND_DESCR("uri", "Build URI");
+	PRINT_MODULE_USAGE_COMMAND_DESCR("px4io_pres", "PX4IO MCU present");
```

Comment:
I'd just call it `px4io_present` so that a reader can understand it. Otherwise, is it a press, pressure, preset?

Reply:
Will do